test/load/test_load_tx_ethbridge_burn.py

```diff
@@ -216,7 +221,7 @@ def _test_load_tx_ethbridge_lock_burn(ctx: test_utils.EnvCtx, amount_per_tx: int
             signed_tx = ctx.sifnode_client.sign_transaction(tx, sif_acct, sequence=sequence,
                 account_number=account_number)
             encoded_tx = ctx.sifnode_client.encode_transaction(signed_tx)
-            rowan_after = ctx.get_sifchain_balance(sif_acct)[rowan]
+            rowan_after = ctx.get_sifchain_balance(sif_acct)["rowan"]
```

Comment:
```
rowan_after = ctx.get_sifchain_balance(sif_acct).get(ROWAN, 0)
```

This will ensure that you will get 0 even if balances don't have a key for rowan.